sklearn/ensemble/_stacking.py

```diff
@@ -0,0 +1,654 @@
+                if (self.predict_method_[est_idx] == 'predict_proba' and
+                        preds.shape[1] == 2):
+                    X_meta.append(preds[:, [1]])
+                else:
+                    X_meta.append(preds)
+        return np.concatenate(X_meta, axis=1)
+
+    def fit(self, X, y, sample_weight=None):
+        """Fit the estimators.
+
+        Parameters
+        ----------
+        X : {array-like, sparse matrix}, shape (n_samples, n_features)
+            Training vectors, where n_samples is the number of samples and
+            n_features is the number of features.
+
+        y : array-like, shape (n_samples,)
+            Target values.
+
+        sample_weight : array-like, shape (n_samples,) or None
+            Sample weights. If None, then samples are equally weighted.
+            Note that this is supported only if all underlying estimators
+            support sample weights.
+
+        Returns
+        -------
+        self : object
+        """
+
+        self._validate_meta_estimator()
+
+        if self.estimators is None or len(self.estimators) == 0:
+            raise AttributeError(
+                "Invalid 'estimators' attribute, 'estimators' should be a list"
+                " of (string, estimator) tuples."
+            )
+
+        names, estimators_ = zip(*self.estimators)
+        self._validate_names(names)
+
+        has_estimator = False
+        for est in estimators_:
+            if est not in (None, 'drop'):
+                has_estimator = True
+
+        if not has_estimator:
+            raise ValueError(
+                "All estimators are None or 'drop'. At least one is required "
+                "to be an estimator."
+            )
+
+        if isinstance(self.predict_method, str):
+            if self.predict_method != 'auto':
+                raise AttributeError(
+                    "When 'predict_method' is a string, it should be 'auto'. "
+                    " Got {} instead.".format(self.predict_method)
+                )
+            predict_method = [self.predict_method] * len(estimators_)
+        else:
+            if len(self.estimators) != len(self.predict_method):
+                raise AttributeError(
+                    "When 'predict_method' is a list, it should be the same "
+                    "length as the list of estimators. Provided {} methods "
+                    "for {} estimators."
+                    .format(len(self.predict_method), len(self.estimators))
+                )
+            predict_method = self.predict_method
+
+        # Fit the base estimators on the whole training data. Those
+        # base estimators will be used in transform, predict, and
+        # predict_proba. They are exposed publicly.
+        self.estimators_ = Parallel(n_jobs=self.n_jobs)(
+            delayed(_parallel_fit_estimator)(clone(est), X, y, sample_weight)
+            for est in estimators_ if est not in (None, 'drop')
```

Comment:
Hmm... this differs from ColumnTransformmer, where the list of fitted transformers includes 'drop's